projects/demo/project.json

```diff
@@ -206,7 +206,7 @@
             "builder": "@nrwl/workspace:run-commands",
             "options": {"command": "nx serve --configuration shadow"}
         },
-        "serve:ssr": {
+        "serve-ssr": {
```

Comment:
what do you think?

<img width="483" alt="image" src="https://user-images.githubusercontent.com/12021443/183928363-e0527d0c-71c0-48bd-b63c-056a966e4e04.png">

Reply:
I have modified separator of all many-words commands to `-` (to be similar to the example from [official angular documentation](https://angular.io/guide/universal)).